data/json/recipes/armor/other.json

```diff
@@ -95,6 +95,30 @@
     "type": "recipe",
     "components": [ [ [ "silver_small", 15 ] ] ]
   },
+  {
+    "result": "bead_bracelet",
+    "category": "CC_ARMOR",
+    "subcategory": "CSC_ARMOR_OTHER",
+    "skill_used": "fabrication",
+    "difficulty": 0,
```

Comment:
```suggestion
```

Reply:
I was getting an error when I set difficulty to 0.  I can try again...

Reply:
removed line